### PR TITLE
EZP-30503: Change names of parameters

### PR DESCRIPTION
**Pull request by pawbuj**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30503](https://jira.ez.no/browse/EZP-30503)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/create/create.html.twig [hunk 1]
```diff
@@ -8,23 +8,23 @@
...
+            <svg class="ez-icon ez-icon-{{ content_type.identifier }}">
+                <use xlink:href="{{ ez_content_type_icon(content_type.identifier) }}"></use>
             </svg>
-            {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
+            {{ 'new_content_item'|trans({'%contentType%': content_type.name})|desc('New %contentType%') }}
         </h1>
 
         <div class="small">
-            {{ contentType.name }} / Parent Location ID: {{ parentLocation.id }}
+            {{ content_type.name }} / Parent Location ID: {{ location.id }}
```
Comment (mikadamczyk) [inline src/bundle/Resources/views/content/create/create.html.twig]: I am against changing `parent location` to `location` in this context. Below we are good with `parent_location`

File: src/bundle/Resources/views/content/create/create.html.twig [hunk 2]
```diff
@@ -39,35 +39,36 @@
...
         </div>
+
     </section>
 {% endblock %}
 
 {% block right_sidebar %}
     {% set content_create_sidebar_right = knp_menu_get('ezplatform_admin_ui.menu.content_create.sidebar_right', [], {
-        'parentLocation': parentLocation,
-        'content_type': contentType,
+        'parent_location': location,
```
Comment (mikadamczyk) [inline src/bundle/Resources/views/content/create/create.html.twig]:
```suggestion
        'parent_location': parent_location,
```

**Comment by lserwatka** [comment]
@pawbuj we should keep `pagelayout` name here

File: src/bundle/Resources/views/account/forgot_password/mail.html.twig [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 {%- block body -%}
     <p>
-        {{ 'ezplatform.forgot_user_password.message'|trans({ '%reset_password%': url('ezplatform.user.reset_password', {'hashKey': hashKey}) })|raw
+        {{ 'ezplatform.forgot_user_password.message'|trans({ '%reset_password%': url('ezplatform.user.reset_password', {'hashKey': has_key}) })|raw
```
Comment (ViniTou) [inline src/bundle/Resources/views/account/forgot_password/mail.html.twig]: It should be `hash_key`, right?

File: src/bundle/Resources/views/content/create/create.html.twig (hunk 1 above)
Comment (pawbuj) [reply src/bundle/Resources/views/content/create/create.html.twig]: https://github.com/ezsystems/repository-forms/blob/89d28cdea7e1639193a92c6eee929172964381d3/lib/Content/View/Builder/ContentCreateViewBuilder.php#L121 

there was a @todo to rename it

Comment (pawbuj) [reply src/bundle/Resources/views/content/create/create.html.twig]: @Nattfarinn, I think you added this @todo comment. Also @webhdx share your opinion, please.

**Pull request merged**